crates/wasm-backend-traits/src/lib.rs

```diff
@@ -85,6 +85,10 @@ pub trait WasmBackend: Clone + Default + 'static {
     /// Creates a new wasm backend with default configuration. In future, a configuration
     /// may be passed as argument.
     fn new() -> WasmBackendResult<Self>;
```

Comment:
I will rethink/rewrite backend creation methods. It's because wasmtime does not allow `call_async` in sync stores. So I either need to completely remove all sync options, or add some kind of sync API throughout the codebase. I like the first option moore